gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileSystem.java

```diff
@@ -113,13 +132,20 @@ public CloudStoragePath getPath(String first, String... more) {
   }
 
   /**
-   * Does nothing.
+   * Does nothing currently. This method <i>might</i> be updated in the future to close all channels
+   * associated with this file system object. However it's unlikely that even then, calling this
+   * method will become mandatory.
    */
   @Override
-  public void close() {}
+  public void close() throws IOException {
+    // TODO(jean-philippe-martin,jart): Synchronously close all active channels associated with this
```

Comment:
Optional: instead of leaving a TODO that probably won't be tracked or changed, you could create a Github issue on gcloud-java.